lib/apple/index.js

```diff
@@ -39,7 +46,7 @@ function parseResult(result) {
 
 	var latestReceiptInfo = null;
 
-	if (status !== 0) {
+	if (status !== 0 && status !== 21006) {
```

Comment:
I'm not able to reproduce a 21006 from the receipts we have records of. Went through quite a few. I'm guessing this happens for one-time purchases than purchases with a terms of renewal. 

I would feel more comfortable with this if we have more actors reproducing the issue.

That being said, this looks fine to me.

Reply:
From Apple's docs:

> This receipt is valid but the subscription has expired. When this status code is returned to your server, the receipt data is also decoded and returned as part of the response.
> Only returned for iOS 6 style transaction receipts for auto-renewable subscriptions.

This is a behavior change, that may make a lot of sense (a bit of a shame that didn't get its own PR though). @KLVTZ Do you have an subscription receipt that matches the description above?

Reply:
@ronkorving Unfortunately we do not. It looks like iOS 6 came out in September 2012. We didn't launch our product until 2015/2016. The odds of us having one are slim to none.

Reply:
Oh, this did not mean *since* iOS6?

Reply:
I think it means since iOS6. I ran across this issues during my testing. My receipts were valid but it was throwing errors when they had expired. That's why I fixed it here.